lib/gitlab/ci/templates/jobs/gradle/DependencyScanning.yml

```diff
@@ -24,7 +24,13 @@ dependency_scanning_validation:
     - ls
```

Comment:
Thanks and I think it does. I probably should have linked that ticket to this branch and added more notes to those issues since they are basically just placeholders at the moment.